src/element-observation.js

```diff
@@ -60,6 +60,15 @@ export class StyleObserver {
     return this.element.style.cssText;
   }
 
+  _setProperty(style, value) {
+    let priority = null;
```

Comment:
`priority` should be empty string (`''`) or `'important'`. [See step 5 from spec](https://www.w3.org/TR/cssom-1/#dom-cssstyledeclaration-setproperty).